src/landscape.jl

```diff
@@ -29,8 +29,8 @@ Creates a landscape of size `dims` (a tuple of two integers) following the model
 defined by `alg`. The `mask` argument accepts a matrix of boolean values, and is
 passed to `mask!` if it is not `nothing`. 
 """
-function Base.rand(alg::T, dims::Tuple{Int64,Int64}; mask=nothing) where {T <: NeutralLandscapeMaker}
-    ret = Matrix{Float64}(undef, dims...)
+function Base.rand(alg::T, dims::Tuple{Int,Int}; mask=nothing) where {T <: NeutralLandscapeMaker}
+    ret = Matrix{AbstractFloat}(undef, dims...)
```

Comment:
Returning a Matrix of AbstractFloats seems like a very expensive price to pay for 32-bit compatibility, as it has no defined memory layout. Maybe instead inquire `Sys.WORD_SIZE` and change accordingly?

Reply:
won't have time to do it in the next future, but definitely